common/src/main/java/com/wynntils/core/WynntilsMod.java

```diff
@@ -85,24 +87,31 @@ private static void handleExceptionInEventListener(Throwable t, Event event) {
             return;
         }
 
+        // TODO: Look into the benefits of crashing non-user features
         if (!(featureOptional.get() instanceof UserFeature feature)) {
             WynntilsMod.error("Exception in event listener in non-user feature: " + crashingFeatureName, t);
             return;
         }
 
-        feature.setUserEnabled(false);
-        feature.tryUserToggle();
+        feature.crash();
 
         WynntilsMod.error("Exception in feature " + feature.getTranslatedName(), t);
         WynntilsMod.warn("This feature will be disabled");
 
-        // TODO: This is a temporary fix for a crash that occurs when an error happens in a client-side message
+        // FIXME: This is a temporary fix for a crash that occurs when an error happens in a client-side message
```

Comment:
@kristofbolyai Btw, is this really temporary? How can we solve this otherwise?

Reply:
I thought about it, not sure atm